lib/govuk_index/elasticsearch_presenter.rb

```diff
@@ -16,28 +16,42 @@ def identifier
     end
 
     def document
+      {}.merge(common_fields)
+        .merge(fields_from_expanded_links)
```

Comment:
@MatMoore exactly, that way it makes sense to test a field at a time - as each field is a method on the object. This hopefully makes the objects cleaner and easier to test.